timesketch/api/v1/resources/event.py

```diff
@@ -1208,3 +1248,167 @@ def post(self, sketch_id):
         )
 
         return HTTP_STATUS_CODE_OK
+
+
+class EventTagResource(resources.ResourceMixin, Resource):
```

Comment:
Can we rename this to EventUnTagResource. Two reasons:
1. We already have the old EventTaggingResource and having this is confusing
2. Move the logic from DELETE to POST. Dealing with delete requests using POST data is not recommended and goes against REST principles. I know that we are doing that in other places but I want to change that going forward.